web-core-sdk/browser-extensions/chrome-extension-web3auth-example/src/App.tsx

```diff
@@ -29,13 +29,10 @@ function App() {
             chainId: "0x13881",
             rpcTarget: "https://rpc.ankr.com/polygon_mumbai",
           },
+          web3AuthNetwork: "cyan",
         });
 
-        const openloginAdapter = new OpenloginAdapter({
-          adapterSettings: {
-            network: "cyan",
-          },
-        });
+        const openloginAdapter = new OpenloginAdapter({});
```

Comment:
Why we're keeping this line @Saviour1001?

Reply:
It's better to remove it. wdyt?

Reply:
removed